README.md

```diff
@@ -46,6 +46,7 @@ you can also set other parameters with the productcat.set method, like country,
 ```js
 productcat.set('app_key', 'YOUR_APP_KEY'); // required.
 productcat.set('cid', "YOUR_CID"); // required.
+productcat.set('tracker_code', 'test')  // code for visenze tracking  
```

Comment:
@swVisenze actually it is not important to migrate tracking to ProductCat SDK. What is important is to update ViSearch Javascript SDK (need to sync with Dashboard release for customers to get the tracking code). For ProductCat, we probably will hold this first as we already have an existing data pipeline.